ui/oopsyraidsy/player_state_tracker.ts

```diff
@@ -549,7 +553,7 @@ export class PlayerStateTracker {
 
     const missedNames = missedIds.map((id) => {
       const name = this.partyTracker.nameFromId(id);
-      if (!name) {
+      if (name === undefined || name.length === 0) {
```

Comment:
```suggestion
      if (name === undefined) {
```

I think we shouldn't bother special-casing empty strings coming from added combatant lines.